CHANGELOG

```diff
@@ -66,7 +66,9 @@ GPU support
   seamlessly with a new lightweight device/host memory manager. The kernels can
   be implemented either in OCCA, or as a simple wrapper around for-loops, which
   can then be dispatched to RAJA and native backends. See the files forall.hpp
-  and mem_manager.hpp in the general/ directory for more details.
+  and mem_manager.hpp in the general/ directory for more details. The memory
+  manager can utilize the Umpire resource management library or handle directly
+  raw CUDA or managed (UVM) pointers.
 
```

Comment:
The `CHANGELOG` changes are in the wrong section: version 4.0 instead of 4.0.1

Reply:
Ok, I'll move them

Reply:
This is not addressed yet!

Reply:
`done`